hack/cherry_pick_pull.sh

```diff
@@ -182,4 +186,10 @@ if ! [[ "${REPLY}" =~ ^[yY]$ ]]; then
 fi
 
 git push origin -f "${NEWBRANCHUNIQ}:${NEWBRANCH}"
+
+if [[ -n "${DRY_RUN}" ]]; then
```

Comment:
It leaves the repository in the same state it's in if you say no to pushing the branch.  I'm actually going to change this a little bit so that you don't push the branch.  Don't merge for now.